lib/Cake/Network/Email/SmtpTransport.php

```diff
@@ -369,7 +369,7 @@ protected function _smtpSend($data, $checkCode = '250') {
 				if ($code[2] === '-') {
 					continue;
 				}
-				return $code[1];
+				return;
```

Comment:
This will break [this line of code](https://github.com/cakephp/cakephp/blob/2.7-return-value/lib/Cake/Network/Email/SmtpTransport.php#L195).